databricks/sdk/service/jobs.py

```diff
@@ -58,8 +58,8 @@ def from_dict(cls, d: Dict[str, any]) -> BaseJob:
 class BaseRun:
     attempt_number: Optional[int] = None
     """The sequence number of this run attempt for a triggered job run. The initial attempt of a run
-    has an attempt_number of 0\. If the initial run attempt fails, and the job has a retry policy
-    (`max_retries` \> 0), subsequent runs are created with an `original_attempt_run_id` of the
+    has an attempt_number of 0. If the initial run attempt fails, and the job has a retry policy
```

Comment:
I would suggest just going with databricks' version here to avoid merge conflicts down the line